spec/tapioca/cli/dsl_spec.rb

```diff
@@ -647,6 +647,63 @@ class User; end
           assert_success_status(result)
         end
 
+        it "can be called by path" do
+          @project.write("lib/models/post.rb", <<~RB)
+            require "smart_properties"
+
+            class Post
+              include SmartProperties
+              property :title, accepts: String
+            end
+          RB
+
+          @project.write("lib/models/nested/user.rb", <<~RB)
+            require "smart_properties"
+
+            module Nested
+              class User
+                include SmartProperties
+                property :name, accepts: String
+              end
+            end
+          RB
+
+          @project.write("lib/job.rb", <<~RB)
+            require "smart_properties"
+
+            class User
+              include SmartProperties
+              property :name, accepts: String
+            end
+          RB
+
+          result = @project.tapioca("dsl lib/models")
+
+          assert_equal(<<~OUT, result.out)
+            Loading Rails application... Done
+            Loading DSL compiler classes... Done
+            Compiling DSL RBI files...
+
+                  create  sorbet/rbi/dsl/nested/user.rbi
+                  create  sorbet/rbi/dsl/post.rbi
+
+            Done
+
+            Checking generated RBI files...  Done
+              No errors found
+
+            All operations performed in working directory.
+            Please review changes and commit them.
+          OUT
+
+          assert_empty_stderr(result)
+
+          assert_project_file_exist("sorbet/rbi/dsl/post.rbi")
+          refute_project_file_exist("sorbet/rbi/dsl/user.rbi")
+
+          assert_success_status(result)
+        end
+
```

Comment:
Can we add a test showing what happens when passing a wrong path `tapioca dsl path/to/nowhere.rb`? 🙏